test-suite/Makefile

```diff
@@ -223,7 +223,7 @@ report: summary.log
 # printed for all opened bugs (still active or seems to be closed).
 # For closed bugs that behave as expected, no message is printed
 
-# All files are assumed to have <# of the bug>.v as a name
+# All files are assumed to have bug_<# of the bug>.v as a name
```

Comment:
Well spotted, indeed. The convention on file names changed and this was not updated.